File: src/main/java/com/bombproj/dto/ProjectFileDto.java
package com.bombproj.dto;

import lombok.Data;

import java.io.Serializable;

@Data
public class ProjectFileDto implements Serializable {

    private static final long serialVersionUID = 1L;

    private String name;

    private String sortName;
    private String sortValue;
    private String projectId;
    private String folderId;

    private String creatUserId;

    private Integer page;
    private Integer pageSize;

}

File: src/main/scala/com/bombproj/dto/ProjectFileDto.scala
package com.bombproj.dto

import scala.beans.BeanProperty

@SerialVersionUID(1L)
case class ProjectFileDto() {

    @BeanProperty var name: String = _
    @BeanProperty var sortName: String = _
    @BeanProperty var sortValue: String = _
    @BeanProperty var projectId: String = _
    @BeanProperty var folderId: String = _
    @BeanProperty var creatUserId: String = _
    @BeanProperty var page: Int = _
    @BeanProperty var pageSize: Int = _

}
